setup_ts.py

```diff
@@ -15,6 +15,7 @@
 import json
 import os
 from pathlib import Path
+import platform
```

Comment:
why the changes to this file are merged to the branch doc-refactor?

Reply:
The changes are meant to make the doc clear.
I don't want to discuss in doc that "you need to add --all because blahblah", or "to install on M1 first change line 123 to blahblah".